samplers/composite_sampler_test.go

```diff
@@ -0,0 +1,63 @@
+package samplers
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+type mockSampler struct {
+	sampled bool
+}
+
+func (c *mockSampler) IsSampled(data interface{}) bool {
+	return c.sampled
+}
+
+func newMockSampler(sampled bool) Sampler {
+	return &mockSampler{sampled}
+}
+
+func TestWithNoSamplers(t *testing.T) {
+	cms := NewCompositeSampler(nil, "")
+	assert.False(t, cms.IsSampled(nil))
+
+	var samplers []Sampler
```

Comment:
Lines 25-26 seems not to be used, are they leftover?

Reply:
yes they are